e2e/autoresizer-cluster.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion:  kind.x-k8s.io/v1alpha4
```

Comment:
Would you please remove this patch if it is not required it? Is it correct that the configuration of `volume-stats-agg-period` is not required to pass the e2e test?

Reply:
Sorry, I fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/44/commits/516f4086639e4183386f279db73517849c4b48be